lady_luck.js

```diff
@@ -48,37 +38,29 @@ function d10RefillCheck() {
 	}
 }
 
-function nwodRoll(pool, options) {
+function nwodRoll(pool, options='') {
```

Comment:
You will never submit a `null` value into options.

Reply:
Further down in the code it checked if options existed so I added it as a precaution